### ISSUE TITLE
[BUG] Cannot assign AutomationProperties.Name to list item if list item contains a usercontrol object

### ISSUE THREAD

**Issue by EltonChen123** [microsoft/axe-windows#631]
## Describe the bug
We define a ListBox and define a customized list item which contains a usercontrol to show the app icon, app name, size... When we set AutomationProperties.Name to list item, it change the name of usercontrol, not list item name. The list item name still show the usercontrol class name. How to change the list item name?

**To Reproduce**
 
![image](https://user-images.githubusercontent.com/89568185/130912119-4c0287c4-b138-433b-b57d-e279ece5d509.png)

![image](https://user-images.githubusercontent.com/89568185/130911714-3cf62e76-a4c3-435a-9880-d4bb2ae928a1.png)

![image](https://user-images.githubusercontent.com/89568185/130910933-2c64f309-39e1-40aa-ab61-56a70ec20396.png)

## Expected behavior
The list item name should be changed.

## Actual behavior
The list item name always show the usercontrol class name.

## Screenshots or .GIF
![image](https://user-images.githubusercontent.com/89568185/130912991-aeec6c92-d870-4e91-8f99-b8268cfea209.png)

## Desktop (please complete the following information):

 - OS: 20H2 19042.1165
 - Accessibility Insights for Windows Version: 1.1.1689.1
 - Target Application: PredatorSense
 - Target Application Version: 

## Additional context

Priority requested -

**Comment by dbjorge** [comment]
Hi @EltonChen123 ,

This is more of a WPF question than an axe-windows bug, so I'm going to mark this issue as closed/out-of-scope for axe-windows.

That said, [this sample project](https://github.com/guybark/WPFListItemAccessibleNames) (particularly, [this bit of XAML](https://github.com/guybark/WPFListItemAccessibleNames/blob/c676611e7984de694ea4e601462c54c926cd5b77/WPFListItemAccessibleNames/MainWindow.xaml#L14-L46)) demonstrates how to do what you're looking for. Let us know if that gets you unblocked!

Thanks and good luck!
